libs/base/ui/navigation/navigation-item/src/navigation-item.component.ts

```diff
@@ -7,6 +7,25 @@ export class NavigationItemComponent extends LitElement {
 
   @property() icon?: string;
 
+  connectedCallback(): void {
+    super.connectedCallback();
+
+    const parent = this.parentNode as Element;
+    const mutationObserver = new MutationObserver(() => {
+      if (parent.hasAttribute('collapsed')) {
+        this.setAttribute('parent-collapsed', '');
+      } else {
+        this.removeAttribute('parent-collapsed');
+      }
+    });
+
+    mutationObserver.observe(parent, { attributes: true });
+
+    if (parent.hasAttribute('collapsed')) {
+      this.setAttribute('parent-collapsed', '');
+    }
+  }
```

Comment:
Wouldn't it be possible to do this through css vars? So then we don't need the mutationObserver.

Something like 
```
:host([collapsed]) {
--transform: translateY(var(--oryx-navigation-item-divider-position)) translateX(-33%);
}
```

```
:host([divider]::after {
transform: var(--transform, none);
}
```

Reply:
I believwe don't need MutationObserver, see proposal at styles